src/main/java/org/jenkinsci/test/acceptance/controller/LocalController.java

```diff
@@ -209,12 +211,24 @@ protected void onReady() throws IOException {
     @Override
     public void stopNow() throws IOException {
         LOGGER.info("Stopping Jenkins (" + getLogId() + ") in " + this);
-        process.getProcess().destroy();
+        if (!process.getProcess().supportsNormalTermination() && SystemUtils.IS_OS_WINDOWS) {
+            new WinProcess(process.getProcess()).sendCtrlC();
+        } else {
+            // https://bugs.openjdk.org/browse/JDK-5101298
+            // process.destroy closes all the streams even if there if the process is still alive and writing
+            // causing shutdown logs to be lost.  sending the signal via the ProcessHandle solves this issue.
+            try {
+                process.getProcess().toHandle().destroy();
+            } catch (UnsupportedOperationException huh) {
+                process.getProcess().destroy();
+            }
+        }
         Runtime.getRuntime().removeShutdownHook(shutdownHook);
         try {
             if (!process.getProcess().waitFor(time.seconds(20), TimeUnit.MILLISECONDS)) {
                 throw new IOException("Jenkins (" + getLogId() + ") failed to stop within the allowed timeout");
             }
+            logWatcher.close();
```

Comment:
this closes the stream.  The logwatcher should have already reached the EOF but this will release the resource.